modules/http2/src/main/java/org/glassfish/grizzly/http2/DefaultOutputSink.java

```diff
@@ -25,7 +25,6 @@
 import java.util.List;
```

Comment:
Grizzly's tests were failing because of this class. I was able to fix one of two failing tests, but then I noticed that the lock usages are not always correct and is hard to find all paths. So I did around 100 small iterations to make the code more readable and to fix both tests.